### DISCUSSION TITLE
No calories imported?

### DISCUSSION THREAD

**Discussion by GShock112** [mytourbook/mytourbook#1046] [Ideas]
Took a log from RGT Cycling and I uploaded it into MTB but I can't see the calories. Is it me, am I missing some setting?

**Comment by FJBDev** [comment]
Can you please attach a screenshot and give more details ? Is it a FIT file ? tcx ? gpx ?

The best would be to attach an example of file for which you encounter the issue so we can reproduce it and investigate it.

**Reply by GShock112** [reply]
I uploaded both the gpx and the fit file here: https://we.tl/t-nVHnxsimMS

Let me know all the tests you want me to do, no problem.

**Reply by FJBDev** [reply]
> I uploaded both the gpx and the fit file here: https://we.tl/t-nVHnxsimMS
> 

Thanks. 
Several things here:
- For the GPX file, per the [GPX schema,](https://www.topografix.com/gpx/1/1/gpx.xsd) the GPX format just doesn't support calories, and looking in the GPX file, you can see that there is no calories data.

- For the FIT file, it seems that the calories doesn't exist in the FIT file. I attached the converted CSV format of the FIT file and as you can see, it's not part of the file.

I would recommend that you contact RGT so that they include the calories in the FIT file.

[RGTCycling_Borrego-Springs_2023-03-30_08-55-00.csv](https://github.com/mytourbook/mytourbook/files/11113075/RGTCycling_Borrego-Springs_2023-03-30_08-55-00.csv)

**Reply by GShock112** [reply]
Thanks FJB I submitted the request to the RGT devs. Will let you know.
